pkg/params/params.go

```diff
@@ -363,6 +363,10 @@ func (p *Param) AsString() string {
 	return p.value
 }
 
+func (p *Param) AsBytes() []byte {
```

Comment:
Please add a test for it in https://github.com/inspektor-gadget/inspektor-gadget/blob/cd4bc3599a8d389f73cea4498d97ec822beab897/pkg/params/params_test.go#L25.